src/pages/heroSection/index.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { Box, Button, Typography } from "@material-ui/core";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+
+import { heroStyles, heroMdStyles, heroSmStyles } from "./styles";
+import { ImagesCarousel } from "./ImagesCarousel";
+
+const imagesList = [
+  {
+    background: "/img/cards-icons/EOS-Previews-2.webp",
+  },
+  {
+    background: "/img/cards-icons/Hero-1.webp",
+  },
+  {
+    background: "/img/cards-icons/Hero-2.webp",
+  },
+];
+
+export const HeroSection = () => {
+  const xsDown = useMediaQuery("(max-width:400px)");
+  const smDown = useMediaQuery("(max-width:600px)");
+  const mdDown = useMediaQuery("(max-width:900px)");
+  const {
+    button,
+    textButton,
+    textTitleImage,
+    dot,
+    textSubtitleImage,
+    textImage,
+  } = smDown ? heroSmStyles() : mdDown ? heroMdStyles() : heroStyles();
+
+  return (
+    <Box width="100%" position="relative">
+      {imagesList.map(({ background }, index) => (
+        <ImagesCarousel
+          key={background}
+          background={background}
+          index={index}
+        />
+      ))}
+      <Box
+        display="flex"
+        justifyContent="center"
+        width="100%"
+        position="absolute"
+        bottom={mdDown ? "160px" : "20vw"}
+      >
+        <Typography className={textTitleImage}>EOS Costa Rica</Typography>
+      </Box>
+      <Box
+        display="flex"
+        justifyContent="center"
+        width="100%"
+        position="absolute"
+        bottom={xsDown ? "140px" : mdDown ? "136px" : "17vw"}
+      >
+        <Typography className={textSubtitleImage}>Developers Guide</Typography>
+      </Box>
+      <Box
+        display="flex"
+        justifyContent="center"
+        width="100%"
+        position="absolute"
+        bottom={mdDown ? "95px" : "12vw"}
+      >
+        <Typography className={textImage}>
+          Welcome to our open-source developers portal. Begin your journey to
```

Comment:
lets include **web3** as an additional keyword here , like this:

Welcome to our web3 developers portal...

Reply:
Ready